Dockerfile

```diff
@@ -1,4 +1,8 @@
 FROM hashicorp/packer:light
 
+RUN packer plugins install github.com/hashicorp/amazon v1.1.0
+
 RUN apk add --no-cache --update python3 jq make aws-cli py3-pip
 RUN pip3 install --no-cache boto3
+
```

Comment:
I would probably write a comment like:
```
# given the plugin github.com/hashicorp/amazon uses directory /tmp, it is best to avoid this directory when using this image.